js/src/logger.ts

```diff
@@ -816,51 +759,8 @@ export async function init(
     isPublic,
     metadata,
   });
-}
-
-/**
- * Wrapper over `braintrust.init`, which passes the initialized `Experiment` it to the given callback and closes it afterwards. See `braintrust.init` for full details.
- *
- * @param options.setCurrent If true (default), set the currently-active experiment to the newly-created one. Equivalent to calling `braintrust.withCurrent(experiment, callback)`.
- */
-export async function withExperiment<R>(
-  project: string,
-  callback: (experiment: Experiment) => R,
-  options: Readonly<InitOptions & SetCurrentArg> = {}
-): Promise<R> {
-  const experiment = await init(project, options);
-  return runFinally(
-    () => {
-      if (options.setCurrent ?? true) {
-        return withCurrent(experiment, () => callback(experiment));
-      } else {
-        return callback(experiment);
-      }
-    },
-    () => experiment.close()
-  );
-}
-
-/**
- * Wrapper over `braintrust.initLogger`, which passes the initialized `Logger` it to the given callback and closes it afterwards. See `braintrust.initLogger` for full details.
- *
- * @param options.setCurrent If true (default), set the currently-active logger to the newly-created one. Equivalent to calling `braintrust.withCurrent(logger, callback)`.
- */
-export async function withLogger<R>(
```

Comment:
maybe we can deprecate this, and just assign to the new globla variable if someone does it?